analysis/technical-analysis.md

```diff
@@ -0,0 +1,474 @@
+components are out-of-scope (see below), attacks that would compromise them through “gross”
+changes such as adding debugging ports are considered to be in scope.
+
+The Confidential Computing Consortium also believes that there exist opportunities to provide guidance to
+those designing, implementing and operating workloads around which types of applications which may be
+more vulnerable to attacks than others, and also issues around lifecycle management to help mitigate attacks.
+
+**5.2.2 Out-of-Scope**
+
+Threat vectors which are generally considered to be out-of-scope for Confidential Computing include:
+
+* **Sophisticated physical attacks**: these are physical attacks that typically require long-term and/or
+invasive access to hardware, including chip scraping techniques and electron microscope probes.
+* **Upstream hardware supply-chain attacks**: these exclude attacks on components of a host system
+that is not directly providing TEE capabilities, but do include attacks on, for instance, a CPU. Examples
+include attacks at chip manufacturing time and attacks at key injection/generation time.
+* **Availability attacks (such as DoS or DDoS) attacks**: not addressed by current hardware-based TEEs
+as part of their threat model. Software projects and service providers may provide mitigations to such
+attacks. (Further discussion of the threat model is provided in section 5.)
+
+### 5.3 Side-Channels
+
+**5.3.1 Background**
+
+TEEs provide a level of assurance for data confidentiality as per the threat model goal described in
+section 5.1. However, this assurance rests on some assumptions, a key one being that there are no
+exploitable side-channels the owner (or any other entities with access to the system) could use to infer
+information about the data or execution. Over the last few years, academic researchers have identified
+and demonstrated vulnerabilities in the design of certain TEEs that allow for side-channel attacks that were
+hitherto considered only theoretical. These mirror similar discoveries in other technologies such as TPMs
+and HSMs, as well as in other isolation or data protection mechanisms. These attacks have caused a lot of
+concern across the technology and security community.
+
+**5.3.2 Example**
+
+Here we give a simplified example from elliptic curve cryptography (ECC). ECC uses elliptic curve point
+multiplication to produce a one-way function. In ECC, multiplying a point P on a curve by *n* results in a new
+point on the curve, *Q*. ECC relies on the fact that it is intractable to determine *n* given both *P* and *Q*. A TEE
+might implement this multiplication using a ‘double-and-add’ method to calculate *Q*. In simplified terms,
+the method iterates through the binary representation of *n* one bit at a time and performs a *doubling*
+operation for each *0* bit and a *doubling* and *addition* operation for each *1* bit.
```

Comment:
Just a note: GitHub supports [MathJax](https://www.mathjax.org), so we could get fancier rendering using LaTeX style `$P$` instead of `*P*`.